spec/schemas/MerkleTreeLeaf.yml

```diff
@@ -0,0 +1,25 @@
+type: object
+description: Merkle tree leaf node.
+required:
+  - type
+  - path
+  - encodedPath
+  - nibbleCount
+  - value
+  - leafHash
+properties:
+  type:
+    description: Merkle tree node type.
+    $ref: "./MerkleTreeNodeTypeEnum.yml"
```

Comment:
It's nice to have these `description`s but I think they will be ignored due to #233.

Reply:
We will need to revisit the descriptions next to $ref